gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+GNOME2_LA_PUNT="yes"
+
+inherit gnome2 toolchain-funcs optfeature
```

Comment:
please sort them (of those, only gnome2 is exporting phase functions, so you are safe).